### PR TITLE
ClientAI - Find the Ip corresponding to the hostname

### PR DESCRIPTION
**Pull request by 31Nathan**
With the parameter, search in the /etc/hosts file to find the corresponding Ip
Raise a RunTimeError if the Ip isn't found

### CODE REVIEW

File: src/zappy_ai_src/zappy_ai [hunk 1]
```diff
@@ -22,6 +22,19 @@ def needHelp(ac : int, av) ->  bool:
             return True
     return False
 
+def getIPFromHostname(hostname : str) -> str:
+    """ Find the IP corresponding to the Hostname, raise an RunTimeError if there isn't """
+    Ip = None
+    with open("/etc/hosts") as file:
```
Comment (Cavonstavant) [inline src/zappy_ai_src/zappy_ai:28]: no

**Review by Cavonstavant — CHANGES_REQUESTED** [review]

File: src/zappy_ai_src/zappy_ai (hunk 1 above)
Comment (Cavonstavant) [reply src/zappy_ai_src/zappy_ai:28]: You should use functions that does the resolving for you. Not doing it by hand

Comment (pariciadra) [reply src/zappy_ai_src/zappy_ai:28]: Can u explain savon pls

File: src/zappy_ai_src/zappy_ai [hunk 2]
```diff
@@ -22,6 +23,10 @@ def needHelp(ac : int, av) ->  bool:
             return True
     return False
 
+def getIPFromHostname(hostname : str) -> str:
+    """ Find the IP corresponding to the Hostname, raise an RunTimeError if there isn't """
+    return gethostbyname(hostname)
```
Comment (Cavonstavant) [inline src/zappy_ai_src/zappy_ai:28]: that's better :+1:

**Review by Cavonstavant — APPROVED** [review]
All good

**Review by GuyomT — APPROVED** [review]
NTR

**Review by pariciadra — APPROVED** [review]
LGTM once again

**Pull request merged**